### PR TITLE
Newsletter : permalink

### PR DESCRIPTION
**Pull request by NicolasEYSSERIC**
Do not forget to check out PR on Components...

### CODE REVIEW

File: core-library/src/main/java/org/silverpeas/core/util/URLUtil.java [hunk 1]
```diff
@@ -405,7 +406,8 @@ public static String getMinifiedWebResourceUrl(final String url) {
     Component(URL_COMPONENT, "/Component/"), Folder(URL_TOPIC, "/Topic/"), File(URL_FILE, "/File/"),
     Document(URL_DOCUMENT, "/Document/"), Version(URL_VERSION, "/Version/"),
     Survey(URL_SURVEY, "/Survey/"), Question(URL_QUESTION, "/Question/"),
-    ForumMessage(URL_MESSAGE, "/ForumsMessage/"), Media(URL_MEDIA, "/Media/");
+    ForumMessage(URL_MESSAGE, "/ForumsMessage/"), Media(URL_MEDIA, "/Media/"),
```
Comment (silveruser) [inline core-library/src/main/java/org/silverpeas/core/util/URLUtil.java]: CRITICAL Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS00115)
CRITICAL Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS00115)

File: core-library/src/main/java/org/silverpeas/core/util/URLUtil.java [hunk 2]
```diff
@@ -405,7 +406,8 @@ public static String getMinifiedWebResourceUrl(final String url) {
...
+    Newsletter(URL_NEWSLETTER, "/Newsletter/");
```
Comment (silveruser) [inline core-library/src/main/java/org/silverpeas/core/util/URLUtil.java]: CRITICAL Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS00115)

**Comment by silveruser** [comment]
SonarQube analysis reported 3 issues
* ![CRITICAL][CRITICAL] 3 critical

Watch the comments in this conversation to review them.

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

**Comment by mmoqui** [comment]
Merged after rebasing master

**Pull request closed without merging**